tests/test_simpleecs.py

```diff
@@ -48,7 +48,7 @@ def test_update(manager):
 def test_system_init_destroy_components(manager):
     counting_system = CountingSystem()
     entity = manager.create_entity()
-    entity.add_component(CountComponent('foo'))
+    entity.add_component(CountComponent(name='foo'))
```

Comment:
Was there a reason to make this change other than "explicit is better than implicit"?